olm-catalog/serverless-operator/serverless-operator.package.yaml

```diff
@@ -4,4 +4,4 @@ channels:
 - name: techpreview
   currentCSV: serverless-operator.v1.5.0
 - name: preview-4.3
```

Comment:
I didn't really follow the discussions around channels recently, but do we need a GA channel here?

Reply:
In a separate PR, yes. It's a JIRA on its own.